apps/aeoracle/src/aeoracle.app.src

```diff
@@ -0,0 +1,16 @@
+{application, aeoracle,
+ [{description, "AE Oracle"},
+  {vsn, "0.1.0"},
+  {registered, []},
+  {applications,
+   [kernel,
+    stdlib,
+    lager,
+    crypto
+   ]},
```

Comment:
Now, `aeoracles` has code dependencies against both `aecore` and `aeutils`, which is a bit unfortunate, but I assume outside the scope of this PR. The dependency to `aecore` is circular (at least from a static perspective), and `aeutils` is not a runnable app, so I guess this list is correct.

Reply:
The circularity to aecore is unavoidable unless we merge the applications.

The aeutils should probably be accessed from all apps, but preferably not call any other app.